commit 710f5d627a98e86f821aceb840b8f2f1fcc6cf75
Merge: 5627ecb8374a 46d2c20b0b10
Author: Linus Torvalds <torvalds@linux-foundation.org>
Date:   Sat Mar 26 13:08:25 2022 -0700

    Merge tag 'usb-5.18-rc1' of git://git.kernel.org/pub/scm/linux/kernel/git/gregkh/usb
    
    Pull USB/Thunderbolt updates from Greg KH:
     "Here is the big set of USB and Thunderbolt changes for 5.18-rc1.
    
      Nothing major in here, just lots of little improvements and cleanups
      and new device support. Highlights are:
    
       - list iterator fixups for when we walk past the end of the list (a
         common problem that was cut/pasted in almost all USB gadget
         drivers)
    
       - xen USB driver "hardening" for malicious hosts
    
       - xhci driver updates and fixes for more hardware types
    
       - xhci debug cable fixes to make it actually work again
    
       - usb gadget audio driver improvements
    
       - usb gadget storage fixes to work with OS-X
    
       - lots of other small usb gadget fixes and updates
    
       - USB DWC3 driver improvements for more hardware types
    
       - Lots of other small USB driver improvements
    
       - DTS updates for some USB platforms
    
      All of these have been in linux-next for a while with no reported
      issues"
    
    * tag 'usb-5.18-rc1' of git://git.kernel.org/pub/scm/linux/kernel/git/gregkh/usb: (172 commits)
      usb: gadget: fsl_qe_udc: Add missing semicolon in qe_ep_dequeue()
      dt-bindings: usb: mtk-xhci: add compatible for mt8186
      usb: dwc3: Issue core soft reset before enabling run/stop
      usb: gadget: Makefile: remove ccflags-y
      USB: usb-storage: Fix use of bitfields for hardware data in ene_ub6250.c
      usb: gadget: eliminate anonymous module_init & module_exit
      usb: usbip: eliminate anonymous module_init & module_exit
      xen/usb: harden xen_hcd against malicious backends
      usb: dwc3: gadget: Wait for ep0 xfers to complete during dequeue
      usb: dwc3: gadget: move cmd_endtransfer to extra function
      usb: dwc3: gadget: ep_queue simplify isoc start condition
      xen/usb: don't use arbitrary_virt_to_machine()
      usb: isp1760: remove redundant max_packet() macro
      usb: oxu210hp-hcd: remove redundant call to max_packet() macro
      usb: common: usb-conn-gpio: Make VBUS supply completely optional
      USB: storage: ums-realtek: fix error code in rts51x_read_mem()
      usb: early: xhci-dbc: Fix xdbc number parsing
      usb: early: xhci-dbc: Remove duplicate keep parsing
      x86/tsc: Be consistent about use_tsc_delay()
      usb: gadget: udc: s3c2410: remove usage of list iterator past the loop body
      ...

diff --cc arch/arm64/boot/dts/xilinx/zynqmp.dtsi
index 056761c974fd,ba68fb8529ee..c715a18368c2
--- a/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
+++ b/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
@@@ -823,6 -824,8 +822,7 @@@
  				interrupt-parent = <&gic>;
  				interrupt-names = "dwc_usb3", "otg";
  				interrupts = <0 65 4>, <0 69 4>;
+ 				clock-names = "bus_early", "ref";
 -				#stream-id-cells = <1>;
  				iommus = <&smmu 0x860>;
  				snps,quirk-frame-length-adjustment = <0x20>;
  				/* dma-coherent; */
@@@ -849,6 -851,8 +848,7 @@@
  				interrupt-parent = <&gic>;
  				interrupt-names = "dwc_usb3", "otg";
  				interrupts = <0 70 4>, <0 74 4>;
+ 				clock-names = "bus_early", "ref";
 -				#stream-id-cells = <1>;
  				iommus = <&smmu 0x861>;
  				snps,quirk-frame-length-adjustment = <0x20>;
  				/* dma-coherent; */